tensorflow/stream_executor/device_description.h

```diff
@@ -156,8 +200,15 @@ class DeviceDescription {
 
   // Returns the CUDA compute capability if we're running on the CUDA platform.
   // If a CUDA compute capability is not available, the major version will be
-  // zero, and the return value will be false.
-  bool cuda_compute_capability(int *major, int *minor) const;
+  // zero, and the return value of DeviceVersion::is_valid() will be false.
+  //
+  // Returns the AMDGCN ISA version if we're running on the ROCm platform.
+  // Minor version would always be zero.
+  // If an AMDGCN ISA version is not available, the major version will be
+  // zero, and the return value of DeviceVersion::is_valid() will be false.
```

Comment:
* s/would always be/is always/

* s/the return value of DeviceDescription::is_valid() will be false/DeviceDescription::is_valid() will return false/

But even better, perhaps rewrite this comment so that it's not redundant with the comment on `DeviceVersion`?

> Gets the "hardware version" of this device, where we know how.  If we don't know how, the returned DeviceVersion's is_valid() returns false.